osu.Game/Overlays/SettingsPanel.cs

```diff
@@ -54,7 +54,7 @@ public abstract partial class SettingsPanel : OsuFocusedOverlayContainer
 
         public SettingsSectionsContainer SectionsContainer { get; private set; }
 
-        private SeekLimitedSearchTextBox searchTextBox;
+        protected SeekLimitedSearchTextBox SearchTextBox;
```

Comment:
```suggestion
        protected SeekLimitedSearchTextBox SearchTextBox { get; private set; }
```

don't want that exposed protected field to be mutable by subclasses